### PR TITLE
Add Boba Network

### PR DESCRIPTION
**Pull request by boyuan-chen**

### CODE REVIEW

File: data/adapters/boba.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        '0xa9059cbb0000000000000000000000005e7a06025892d8eef0b5fa263fa0d4d2e5c3b54900000000000000000000000000000000000000000000000000038d7ea4c68000',
+    });
+    const totalGasCostWei = l2GasPrice.mul(l2GasEstimate).toNumber();
+    const ethPrice = await sdk.coinGecko.getCurrentPrice('ethereum');
+    return (totalGasCostWei * ethPrice) / 1e18;
+  };
+
+  const getSwapCost = async () => {
+    const l2GasPrice = await provider.getGasPrice();
+    const l2GasEstimate = await provider.estimateGas({
```
Comment (smartcontracts) [inline data/adapters/boba.ts:85]: Does Boba still use the OVM 1.0 fee scheme? Or does it also use the OVM 2.0 scheme where there's an L2 gas price and an L1 gas price?

Comment (smartcontracts) [reply data/adapters/boba.ts:85]: Boba docs claim to be using the OVM 2.0 fee scheme, so the L1 cost is not being accounted for here: https://docs.boba.network/developer-docs/003_fee-scheme-ovm-2.0#for-frontend-and-wallet-developers

Comment (boyuan-chen) [reply data/adapters/boba.ts:85]: We recently added the l1 security fee. The PR was updated.

**Comment by dmihal** [comment]
Closing, as the adapter was added to https://cryptostats.community instead

**Pull request closed without merging**